openslides/motions/config_variables.py

```diff
@@ -237,4 +237,4 @@ def get_config_variables():
         label='Show paragraph numbering (only in PDF)',
```

Comment:
@emanuelschuetze I get a travis error. These are the last lines:
```
File "/home/travis/build/OpenSlides/OpenSlides/openslides/motions/pdf.py", line 28, in motions_to_pdf
    motion_to_pdf(pdf, motion)
  File "/home/travis/build/OpenSlides/OpenSlides/openslides/motions/pdf.py", line 143, in motion_to_pdf
    convert_html_to_reportlab(pdf, motion.text)
  File "/home/travis/build/OpenSlides/OpenSlides/openslides/motions/pdf.py", line 300, in convert_html_to_reportlab
    if config["motions_pdf_paragraph_numbering"]:
  File "/home/travis/build/OpenSlides/OpenSlides/openslides/core/config.py", line 38, in __getitem__
    raise ConfigNotFound(_('The config variable {} was not found.').format(key))
openslides.core.exceptions.ConfigNotFound: The config variable motions_pdf_paragraph_numbering was not found.
```
So the config variable is still somewhere needed. I put it back in and someone who knows the pdf system can remove it.